src/axis.ts

```diff
@@ -212,35 +307,35 @@ export interface AxisConfigMixins {
   /**
    * X-axis specific config.
    */
-  axisX?: VgAxisConfig;
+  axisX?: VgAxisConfigNoSignals;
```

Comment:
If you use `VgAxisConfigNoSignals` here, then the comment won't be Vega-LIte's?

Reply:
I wonder if we should just make all of these just using AxisConfig, and file a bug and shortTimeLabels doesn't work for these yet.  This way, our schema will be way smaller (no two versions of axis config).

Reply:
Yes. I thought that was the idea here. We don't do anything smart with the values anyway, right?

Reply:
Hmm, we should probably use AxisMixins as well. I'll fix it.

Reply:
Hmm, I'm confused what you do mean "I thought that was the idea here."

Reply:
Oh I see. you mean the comment should be different.   The idea is just the axis-specific config does not support shortTimeLabels yet.  But that's partly why our schema is gigantic.  So I'm voting for consolidating them to use the same definition.

Reply:
Actually, we don't support `grid` here, right?